src/controllers/fcm.controller/fcmController.js

```diff
@@ -0,0 +1,18 @@
+const catchAsync = require('../../utils/catchAsync');
+const { UpdateUserFCMToken } = require('./queries/queries');
+const { query: Hasura } = require('../../utils/hasura');
+
+const saveToken = catchAsync(async (req, res) => {
+  const { fcm_token, cognito_sub } = req.body;
+
+  await Hasura(UpdateUserFCMToken, {
+    cognito_sub,
+    fcm_token,
+  });
+
+  res.status(200).json({
```

Comment:
Follow the same response format as we doing for other controller. Ex: https://github.com/Shivz3232/inovact-nodejs-veyga/blob/11d42b5699b5534449b45921666bbff3cebb0910/src/controllers/idea.controller/likeIdea/index.js#L58